src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -219,6 +219,8 @@
 
     /** This property indicates whether disable_mbean_registration is true */
     IS_DISABLED_MBEAN_REGISTRATION("org.apache.cassandra.disable_mbean_registration"),
+    /** This property indicates if the code is running under the in-jvm dtest framework */
```

Comment:
does the order of the enum matter here? Some enums might be sensitive to the order, so usually new enums are added at the end. It might not be the case here

Reply:
Thankfully no - this enum is really just a convenience wrapper around system properties, so the ordering doesn't matter (and other folks seem to add things where they make sense vs. at the end as well).

Reply:
On rebase, it ends up it now needs to be maintained in alphabetical order, so I had to move things around anyway.